providers/standard/src/airflow/providers/standard/operators/trigger_dagrun.py

```diff
@@ -86,8 +85,12 @@ def get_link(self, operator: BaseOperator, *, ti_key: TaskInstanceKey) -> str:
         # stored in xcom during execution of the triggerING task.
         triggered_dag_run_id = XCom.get_value(ti_key=ti_key, key=XCOM_RUN_ID)
 
-        query = {"dag_id": trigger_dag_id, "dag_run_id": triggered_dag_run_id}
-        return build_airflow_url_with_query(query)
+        if AIRFLOW_V_3_0_PLUS:
+            base_url = conf.get_mandatory_value("fastapi", "base_url").lower()
+            return f"{base_url}/webapp/dags/{trigger_dag_id}/runs/{triggered_dag_run_id}"
```

Comment:
@jedcunningham mentioned in Slack that webapp prefix is removed
https://apache-airflow.slack.com/archives/C06K9Q5G2UA/p1740500908318299